lib/kubeclient/common.rb

```diff
@@ -204,6 +204,11 @@ def retrieve_all_entities(entity_types)
         end
       end
 
+      def proxy_url(type, service, port_name, namespace = '')
+        ns_prefix = build_namespace_prefix(namespace)
+        rest_client["proxy/#{ns_prefix}#{type}/#{service}:#{port_name}"].url
```

Comment:
@oschreib let's move the proxy to `/api/v1/namespaces/{namespace}/pods/{name}:{port}/proxy/{path:*` as suggested by @jimmidyson

Leave services as it is (current url) and add a comment to switch services to the uri when ready.